### ISSUE TITLE
making mpy-cross 

### ISSUE THREAD

**Issue by BLCKPSTV** [micropython/micropython#2742]
Making mpy-cross gives me make: *** mpy-cross: No such file or directory.  Stop.

The dependencies where installed correctly without problems. 
Wemos D1 Mini pro 16mb

EDIT: Is this:  make -C crostool-NG ??

**Comment by dpgeorge** [comment]
You need to build mpy-cross from within the mpy-cross directory of this repository.  So you either do `cd mpy-cross` and then `make`, or just `make -C mpy-cross` from the root of this repository.

**Comment by dhylands** [comment]
```
make -C ../../../mpy-cross target
```
(use appropriate number of ..'s)

**Comment by BLCKPSTV** [comment]
So I'm making the micro-python on for the ESP via this [PFalcon](https://github.com/pfalcon/esp-open-sdk) and to flash it on a ESP8266 device, it requires me to build a case-sensitive image. I manage to clone and build the [repository](git clone --recursive https://github.com/pfalcon/esp-open-sdk.git). And even managed to make/build it. 

From that moment I could go back to the build instruction in the subfolder of the ESP8266 folder in this github [here](https://github.com/micropython/micropython/tree/master/esp8266)

git submodule update --init worked. But then the `make -C mpy-cross` doesn't work. So I downloaded the master folder and tried to make the the mpy-cross from that folder. But my thought was, that it has to be in the case sensitive folder. 
But even the `make -C mpy-cross` didn't got me to build it. I got this : `Use make V=1 or set BUILD_VERBOSE in your environment to increase build verbosity.`

I was able to make it so far with little knowledge of python en and the command line. Really stuck at it. 

The `make -C ../../../mpy-cross target` didn't work as well because there is no top laying folder available in the image.
